src/olympia/search/filters.py

```diff
@@ -175,20 +175,38 @@ def get_value_from_reverse_dict(self):
         return self.get_value_from_object_from_reverse_dict()
 
 
-class AddonTagFilterParam(AddonFilterParam):
+class AddonTagFilterParam(AddonQueryParam):
     # query_param is needed for SearchParameterFilter below, so we need it
     # even with the custom get_value() implementation.
     query_param = 'tag'
 
     def get_value(self):
         return self.request.GET.get(self.query_param, '').split(',')
 
-    def get_es_filter(self):
+    def get_es_query(self):
         # Just using 'terms' would not work, as it would return any tag match
         # in the list, but we want to exactly match all of them.
         return [Q('term', tags=tag) for tag in self.get_value()]
 
 
+class AddonExcludeAddonsExcludeParam(AddonQueryParam):
```

Comment:
Either continue naming this `FilterParam` or `QueryParam` but not just `Param` :)